personalities/ct-static-api/handlers.go

```diff
@@ -256,15 +192,10 @@ func NewCertValidationOpts(trustedRoots *x509util.PEMCertPool, currentTime time.
 	return vOpts
 }
 
-type leafChainBuilder interface {
-	BuildLogLeaf(ctx context.Context, chain []*x509.Certificate, logPrefix string, merkleLeaf *ct.MerkleTreeLeaf, isPrecert bool) (*trillian.LogLeaf, error)
-	FixLogLeaf(ctx context.Context, leaf *trillian.LogLeaf) error
-}
-
 // logInfo holds information for a specific log instance.
 type logInfo struct {
-	// LogPrefix is a pre-formatted string identifying the log for diagnostics
-	LogPrefix string
+	// LogOrigin is a pre-formatted string identifying the log for diagnostics
```

Comment:
Does the comment need a bit of updating?

Reply:
Yeah the code in config.go was unnecessarily complex I think, I've simplified it a bunch, ptal.